parser/src/keys/mod.rs

```diff
@@ -254,6 +267,8 @@ pub fn str_to_oscode(s: &str) -> Option<OsCode> {
         "f22" => OsCode::KEY_F22,
         "f23" => OsCode::KEY_F23,
         "f24" => OsCode::KEY_F24,
+        #[cfg(target_os = "macos")]
```

Comment:
```suggestion
        #[cfg(any(target_os = "macos", target_os = "unknown"))]
```
similar to above (https://github.com/jtroo/kanata/pull/652/files#r1419935217)